packages/react-ui/src/widgets/legend/LegendCategories.js

```diff
@@ -76,6 +85,10 @@ const useStyles = makeStyles((theme) => ({
       boxSizing: 'content-box'
     }
   },
+  icon: {
+    maskRepeat: 'no-repeat',
+    maskSize: 'cover'
+  },
```

Comment:
@VictorVelarde 
So update on story:
 - yes, MUI adds proper prefixes, when used in `makeStyles` like above!
 - but something else (react/mui?) also requires Webkit prefixes ... and filters for inline `style={{...}}`

Reply:
Great, I didn't notice those were inline styles and they were excluded. This way is better, thx!